configs/linux/Agent/Ubuntu/Ubuntu.Dockerfile

```diff
@@ -43,7 +43,7 @@ ENV DOTNET_CLI_TELEMETRY_OPTOUT=true \
 ARG dotnetCoreLinuxComponent
 
 RUN apt-get update && \
-    apt-get install -y git mercurial apt-transport-https ca-certificates software-properties-common && \
```

Comment:
`ca-certificates` was installed in `configs/linux/MinimalAgent/Ubuntu/Ubuntu.Dockerfile`, L#22.